.github/pull_request_template.md

```diff
@@ -7,5 +7,6 @@
 - [ ] I have performed a self-review of my code.
 - [ ] I have included a small demo of the changes. For the UI, this would be a screenshot or a Loom video.
 - [ ] If this is a new feature, I have added unit tests and integration tests.
-- [ ] I have manually run the integration tests and they are passing.
+- [ ] I have run the integration tests locally and they are passing.
 - [ ] All features on the UI continue to work correctly.
+- [ ] I have added the appropriate CI label to the PR.
```

Comment:
@hsubbaraj-spiral could you make this more explicit? Basically repeating what you have in the PR description: skip v.s. run.